easytab.h

```diff
@@ -870,21 +879,41 @@ EasyTabResult EasyTab_Load_Ex(HWND Window,
 
 EasyTabResult EasyTab_HandleEvent(HWND Window, UINT Message, LPARAM LParam, WPARAM WParam)
 {
+    static bool LeftContext = true;
     PACKET Packet = { 0 };
-
     if (Message == WT_PACKET &&
         (HCTX)LParam == EasyTab->Context &&
         EasyTab->WTPacket(EasyTab->Context, (UINT)WParam, &Packet))
     {
         POINT Point = { 0 };
         Point.x = Packet.pkX;
         Point.y = Packet.pkY;
-        ScreenToClient(Window, &Point);
+        if(LeftContext && EasyTab->IsRelative)
+        {
+            Point.x = 0;
+            Point.y = 0;
+            LeftContext = false;
+        }
+        if(!EasyTab->IsRelative)
+        {
+            ScreenToClient(Window, &Point);
+        }
         EasyTab->PosX = Point.x;
         EasyTab->PosY = Point.y;
-
+        EasyTab->Buttons = Packet.pkButtons;
```

Comment:
I think we'll have to store and interpret the button data differently. The documentation states: 

**pkButtons**

In absolute mode, is a DWORD containing the current button state. In relative mode, is a DWORD whose low word contains a button number, and whose high word contains one of the following codes:

| Value |Meaning| 
| ------- | --------- |
|TBN_NONE|no change in button state.| 
|TBN_UP|button was released.|
|TBN_DOWN|button was pressed.|

So we'll need to interpret values differently based on what mode we're in, and need to store them in a readily available format for the user to read.

If this seems like a crazy amount of work to get simple information, the whole purpose of this library is to make a sane API for the user, so we have to take care of all the insanity that is the Wacom API. ;)